DigitalOffice/Services/FileService/Repositories/FileRepository.cs

```diff
@@ -19,8 +20,20 @@ public Guid AddNewFile(DbFile file)
         {
             dbContext.Files.Add(file);
             dbContext.SaveChanges();
-            
+
             return file.Id;
         }
+
+        public DbFile GetFileById(Guid fileId)
+        {
+            var file = dbContext.Files.FirstOrDefault(file => file.Id == fileId);
```

Comment:
parameter 'file' hides outer local variable with the same name. Maybe name of variable should be dbFile(it gives more useful information)

Reply:
Fixed